components/compass_design_provider/index.tsx

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
i didnt understand sorry.

Reply:
OK, maybe you are not aware of it, but we are currently restarting the effort of building a component library ... this time it will be based on Material UI and since we will handle a lot of components with this it would be great to know which components got recently added that mimik that compass design look and feel so we can start replacing those once we reach a point where we have a sufficient amount of components ready

Reply:
@michelengelen i meant where do you want the todo comment to be and what it should include?